code/modules/client/preferences_savefile.dm

```diff
@@ -514,6 +514,8 @@ SAVEFILE UPDATING/VERSIONING - 'Simplified', or rather, more coder-friendly ~Car
 	S["scars3"]							>> scars_list["3"]
 	S["scars4"]							>> scars_list["4"]
 	S["scars5"]							>> scars_list["5"]
+	if(length(S["modified_limbs"]))
+		safe_json_decode(S["modified_limbs"])					>> modified_limbs
```

Comment:
```suggestion
	var/limbmodstr
	S["modified_limbs"]													>> limbmodstr
	if(length(limbmodstr))
		modified_limbs = safe_json_decode(limbmodstr)
	else
		modified_limbs = list()
```
also you might want to sanitize it in sanitization proc too, with an equally runtime-safe setup

Reply:
(this ensures if the saved data is empty/null they get their limbs wiped instead of any potentially old data lingering)